book/src/faq.md

```diff
@@ -170,6 +172,27 @@ ERRO Failed to download attester duties      err: FailedToDownloadAttesters("Som
 
 This means that the validator client is sending requests to the beacon node. However, as the beacon node is still syncing, it is therefore unable to fulfil the request. The error will disappear once the beacon node is synced. 
 
+### <a name="bn-fork-choice"></a> My beacon node logs `WARN Error signalling fork choice waiter`, what should I do?
+
+An example of the full log is shown below:
+
+```
+WARN Error signalling fork choice waiter slot: 6763073, error: ForkChoiceSignalOutOfOrder { current: Slot(6763074), latest: Slot(6763073) }, service: state_advance
+```
+
+This suggests that the computer resources are being overwhelmed. It could be due to high CPU usage or high disk I/O usage. This can happen, e.g., when the beacon node is downloading historical blocks, or when the execution client is syncing. The error will disappear when the resources used return to normal or when the node is synced.
```

Comment:
> Since this is functionally the same as the `ERRO Aggregate attestation queue full` error, for consistency we could directly copy the text for that section.

Do you mean having the same content of answers for these two questions?

Reply:
Yep that's right. I think we may as well use the same answer for both

Reply:
Revised as suggested, thank you